### ISSUE TITLE
Tabulation Problem

### ISSUE THREAD

**Issue by Mathie01** [dothinking/pdf2docx#157]
Hello !

I have a summary on a PDF that looks like this:

![image](https://user-images.githubusercontent.com/67562521/182177455-ed31a3e2-35dc-4a85-8427-483a5df8dce7.png)

Problem, when I convert the PDF to docx, I get : 

![image](https://user-images.githubusercontent.com/67562521/182177795-0c14087f-3f12-4d6b-8692-14bbb7285159.png)

The PDF file : 

[manuel.pdf](https://github.com/dothinking/pdf2docx/files/9234857/manuel.pdf)

By the way: a big GG for all this work because it is titanic. A huge thank you!

Thank you 0:)

**Comment by dothinking** [comment]
Sorry for my late response.

In your case, more than one tab stops exist in one paragraph (see below image), so it might need more than one tabs to reach the right position. However, only one tab is considered in current version.

![img](https://user-images.githubusercontent.com/13885442/192135740-ae35130e-295c-4969-b350-a8a4eee024bf.png)

**Comment by dothinking** [comment]
The fix was push to the master branch, but didn't publish a new pypi version yet. So, please install the update directly from this project:

```
$ pip install git+git://github.com/dothinking/pdf2docx.git@master --upgrade
```

**Comment by dothinking** [comment]
Fixed in version `v0.5.7`.
